src/test/resources/feature/TestScenarios.feature

```diff
@@ -0,0 +1,49 @@
+@Claudia
+Feature: Add, remove and modify items from shopping cart
+
+  Scenario: Add items to cart
+
+    Given user is on items page
+    When user adds items with following price
+      | Item price |
+      | $16.51      |
+      | $27.00      |
+    And click on [Add to cart] button
+    Then a pop up window is displayed with message "Product successfully added to your shopping cart"
+    And item details are displayed
+
+  Scenario: Remove items from cart
```

Comment:
If you are starting here (and in the Modify number of items from shopping cart scenario) also from items page (you have added this step in previous one scenario "Given user is on items page") please add this step in Background for all of the scenarios.

Reply:
Done